README.md

```diff
@@ -1,44 +1,19 @@
-# Mental Health Awareness Outreach Event
+# Awesome Books
 
-This is a website that contains information and registration details to a mental health awareness event that is being hosted.
+This is a website that allows the user to add and remove books
```

Comment:
- [ ] Kindly add all the necessary sections from the [README template](https://github.com/microverseinc/readme-template) like `Getting Started`, `Contributing`, `Show your support`, `Acknowledgment` and personalize the contents to make your README descriptive and more informative.
- [ ] Do not forget to add instructions on how to clone your repo and get it working in the terminal
- [ ] Please add the second author's details since this project has two authors.

Reply:
Thank you for the feedback. We have completed all required changes.

Reply:
@julie-ify